infra/modules/worklytics-connectors-msft-365/main.tf

```diff
@@ -60,6 +64,13 @@ module "msft_365_grants" {
   todo_step                = var.todo_step
 }
 
+resource "local_file" "todo-with-external-todo" {
+  for_each = local.todos_to_populate
+
+  filename = module.msft_365_grants[each.key].filename
+  content  = format("%s\n## Setup\nThen, please follow next instructions for complete the setup: \n\n%s", module.msft_365_grants[each.key].todo, each.value.external_token_todo)
```

Comment:
Using this as an override of the current file generated by `msft_365_grants` module, only if the connector has a todo to append. ATM, Teams is the only one having this.